DocAI Incubator Tools/best-practices/HITL_Rejected_Documents_Tracking.ipynb

```diff
@@ -0,0 +1,415 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #109.            hitl_op_id = hitl_path.split("/")[-1]
</pre><p>You don't need to do this, just send hitl_path to the get_operation request</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>